taipy/gui/viselements.json

```diff
@@ -1735,7 +1735,40 @@
                     }
                 ]
             }
-        ]
+        ],
+        [
+            "alert",  
+            {
+                "inherits": ["shared"],
+                "properties": [
+                    {
+                        "name": "severity",
+                        "type": "dynamic(str)",
+                        "default_value": "\"error\"",
+                        "doc": "Defines the severity level of the notification. Valid values: \"error\", \"warning\", \"info\", \"success\"."
+                    },
+                    {
+                        "name": "message",
+                        "default_property": true,
+                        "type": "dynamic(str)",
+                        "default_value": "\"\"",
+                        "doc": "The message displayed in the notification. Can be a dynamic string."
+                    },
+                    {
+                        "name": "variant",
```

Comment:
I'm not sure we need this 'variant' property, although it simplifies styling.
I mean no other element has that kind of styling=-dedicated property.
We will discuss that.